src/core/linter-rules/check-period-in-p.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Linter rule "check-period". Makes sure the there are no periods missing at the end of a <p>
+ *   in the ReSpec config.
+ */
+import { lang as defaultLang } from "core/l10n";
+import LinterRule from "core/LinterRule";
+
+const name = "check-period-in-p";
+
+const meta = {
+  en: {
+    description: "`<p>` tags should end with a period",
+    howToFix: "Please put a period at the end of this `<p>` tag",
+  },
+};
+// Fall back to english, if language is missing
+const lang = defaultLang in meta ? defaultLang : "en";
+
+/**
+ * Runs linter rule.
+ *
+ * @param {Object} config The ReSpec config.
+ * @param  {Document} doc The document to be checked.
+ */
+function lintingFunction(conf, doc) {
+  
+  const offendingElements = Array.from(doc.querySelectorAll("p"))
+    .filter((elem) => !(/[.!?:]$|^ *$/.test(elem.textContent.trim()))
```

Comment:
```JS
// punctuation marks OR zero or more space characters
const punctuationRegExp = /[.!?:]$|^\s*$/m
const offendingElements = Array.from(doc.querySelectorAll("p"))
  .filter(({textContent} ) => !(punctuationRegExp.test(textContent))
```

Reply:
Something like the above... didn't test it.

Reply:
Sure sure. I'll test it and push it right away :+1:

Reply:
Done :grin: :grin: . And I've tested all edge cases too.

Reply:
Keep those edge cases. Will be useful for actual testing.

Reply:
Yeah Sure. Let's work on `{tests: n}` now. Tests are also running, I verified. Also we'll have to introduce new ones for the other punctuations and spaces . I'll include them too. 😁

Reply:
@marcoscaceres any info here?
> Let's work on {tests: n}